msteams-platform/messaging-extensions/build-api-based-message-extension.md

```diff
@@ -8,513 +8,28 @@ ms.author: anclear
 ms.date: 10/19/2023
```

Comment:
doc details are not aligned to the file name.

doc is talking about auth. title is about building api ME,,,, pls fix

Reply:
Updated the meta title